flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -174,6 +174,19 @@
 				" help relieve back-pressure caused by unbalanced data distribution among the subpartitions. This value should be" +
 				" increased in case of higher round trip times between nodes and/or larger number of machines in the cluster.");
 
+	/**
+	 * Number of max backlogs can be used for each output subparition.
+	 */
+	@Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
+	public static final ConfigOption<Integer> NETWORK_MAX_BACKLOGS_PER_SUBPARTITION =
+			key("taskmanager.network.max-backlogs-per-subpartition")
```

Comment:
rename to `max-buffers-per-channel` to be consistent with `buffers-per-channel`?